src/api/config/routes/webui_routes.rb

```diff
@@ -306,7 +306,8 @@
     scope :my do
       resources :tasks, only: [:index], controller: 'webui/users/tasks', as: :my_tasks
 
-      resources :notifications, only: [:index, :update], controller: 'webui/users/notifications', as: :my_notifications
+      resources :notifications, only: [:index], controller: 'webui/users/notifications', as: :my_notifications
```

Comment:
What is the reason behind this change?

Reply:
To the plural `resources` I must pass the id to the path helper, which doesn't work out with the form and the checkboxes

Reply:
But I guess there is a more elegant way, so I will have another look

Reply:
I've found a better way to have this. It's much more explicit:

```suggestion
      resources :notifications, only: [:index], controller: 'webui/users/notifications', as: :my_notifications do
        collection do
          # We allow updating multiple notifications in a single HTTP request
          put :update
        end
      end
```

With this change, the path helper `my_notification_path` will have to be changed to `my_notifications_path`.